numpy/lib/format.py

```diff
@@ -585,7 +585,9 @@ def write_array(fp, array, version=None, allow_pickle=True, pickle_kwargs=None):
         The version number of the format. None means use the oldest
         supported version that is able to store the data.  Default: None
     allow_pickle : bool, optional
-        Whether to allow writing pickled data. Default: True
+        Whether to allow writing pickled data. Default: False
+        Switched from True to False in version 1.17.0 and 1.16.3 in response
+        to CVE-2019-6446.
```

Comment:
This isn't consistent with the release notes, which says it _will_ change. How about a `FutureWarning` in 1.16.3, and a change in 1.17.0?